lib/scala/pkg/src/main/scala/org/enso/pkg/Config.scala

```diff
@@ -85,10 +87,11 @@ object Contact {
 
 /** Represents a package configuration stored in the `package.yaml` file.
   *
-  * @param name package name
-  * @param namespace    package namespace. This field is a temporary workaround
-  *                     and will be removed with further improvements to the
-  *                     libraries system. The default value is `local`.
+  * @param name the package display name
+  * @param module the package module name
```

Comment:
Why do we call the name a _"module"_? As far as I know Enso module is equal a single `.enso` file. Aren't we confusing the terminology?

Wouldn't it be easier to use `displayName` of a project and either `name` or `snakeName` for the snake case name.

Reply:
Where is _module name of a project_ defined? All usages of _module name_ I see in the codebase refer to name of a single `.enso` file!

Reply:
[normalized_name](https://github.com/enso-org/enso/pull/7397#issuecomment-1655270448) addresses my concerns. Let's do it.